src/libs/OptionsListUtils.ts

```diff
@@ -1931,8 +1939,20 @@ function getOptions(
             return;
         }
 
-        if ((!accountIDs || accountIDs.length === 0) && !isChatRoom) {
-            return;
+        if (!accountIDs || accountIDs.length === 0) {
+            if (!isSearchingForReports) {
+                if (!isChatRoom) {
+                    return;
+                }
+            }
+
+            // Rooms that aren't searchable via participant won't have any participants returned with SearchForReports
+            // so we have to let their options pass through this check.
+            // Rooms that are searchable via participant should always have participants returned with SearchForReports,
+            // so we block any that don't have participants with this check.
```

Comment:
>Rooms that are searchable via participant should always have participants returned with SearchForReports,
>so we block any that don't have participants with this check.

It's kind of contradictory? They should always have participant, but we block any that don't have participants? How does that happen exactly?

And is it a different situation from: 

```js
if ((!accountIDs || accountIDs.length === 0) && !isChatRoom) {
    return;
```

Reply:
it sounds like this is the ultimate goal:

>Rooms that aren't searchable via participant won't have any participants returned with SearchForReports so we have to let their options pass through this check

Could we leave the existing code and just do:

```js
if (isSearchingForReports && !ReportUtils.isSearchableViaParticipants(report)) {
    return option;
}
```

and then leave the rest of the code as it is?

Reply:
If a report that should have participants doesn't because of some bug, shouldn't we allow it to be an option anyway?

Reply:
> Could we leave the existing code and just do:
> 
> ```js
> if (isSearchingForReports && !ReportUtils.isSearchableViaParticipants(report)) {
>     return option;
> }
> ```

Are you saying to do that as an early return before we handle do the empty participants check? I think that might also work, we're basically saying if we're searching for reports and the report is not searchable via participant, then we don't really care about checking the report's participants. Let me try that.

Reply:
I think you'll both be much happier with the new stuff :)